TDDPractice/ComplicatedPageTests.cs

```diff
@@ -20,5 +20,18 @@ public void TestMethod1()
             var amazonSearchPage = complicatedPage.SearchUsingAmazon("automation testing");
             Assert.AreEqual("automation testing", amazonSearchPage.ActualSearchResults);
         }
+
+        [TestMethod]
+        public void TestMethodNew()
+        {
+            var path = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
```

Comment:
@andreea-qa instead of relying on a local driver, can we rely on that Nuget package again to get a dynamic chromedriver? That way these tests always work.

Reply:
@nadvolod I changed this to be the same as our other tests